extension/goext/schemas.go

```diff
@@ -267,18 +267,21 @@ type ISchema interface {
 	ColumnNames() []string
 
 	// Properties returns properties of schema
-	Properties() []Property
+	Properties() map[string]Property
 
 	// Extends return list of schema_ids which given schema extends
 	Extends() []SchemaID
 }
 
 // Property represents schema property
 type Property struct {
-	ID       string
-	Title    string
-	Relation SchemaID
-	Type     string
+	ID         string
+	Title      string
+	Relation   SchemaID
+	Type       string
+	Enum       []string
```

Comment:
Just a note. I do not think we need to change this type, but enum is parsed in yaml parser then some of items will not be "string" type (e.g., integer, bool or nil). So, at least unit test should contain the schema yaml file which includes non-string values to check if no error happens.

Reply:
It's a valid point. in fact, one of the test schemas uses ints for enum, this is why I'm converting all values using `fmt.Sprint`.
Let me change the field type to `[]interface{}`and let the clients do the appropriate conversion.